KuringApp/KuringApp/Subscription/SubscriptionView.swift

```diff
@@ -51,74 +81,205 @@ struct SubscriptionFeature: Reducer {
 struct SubscriptionView: View {
     let store: StoreOf<SubscriptionFeature>
     
+    // TODO: 디자인 시스템 분리 - ColorSet
+    struct Constants {
+        static let kuringPrimary: Color = Color(red: 0.24, green: 0.74, blue: 0.5)
+    }
+    
     var body: some View {
         WithViewStore(self.store, observe: { $0 }) { viewStore in
-            List {
-                Text("알림 받고 싶은 카테고리를 선택해 주세요")
+            VStack(alignment: .leading) {
+                HStack {
+                    Text("알림 받고 싶은 \n카테고리를 선택해주세요")
+                        .font(.system(size: 24, weight: .bold))
+                        .foregroundColor(Color(red: 0.1, green: 0.12, blue: 0.15))
+                }
+                .padding(.top, 32)
+                .padding(.bottom, 60)
                 
-                /**
-                 - `viewStore.$subscriptionType`
-                 */
-                Section {
-                    Picker("", selection: viewStore.$subscriptionType) {
-                        Text("일반 카테고리")
-                            .tag(SubscriptionFeature.State.SubscriptionType.university)
-                        
-                        Text("학과 카테고리")
-                            .tag(SubscriptionFeature.State.SubscriptionType.department)
+                LazyVGrid(columns: Array(repeating: .init(.flexible()), count: 2)) {
+                    // TODO: 디자인 시스템 분리 - 칩
+                    VStack {
+                        if viewStore.subscriptionType == .university {
+                            didSelectChipView("일반 카테고리")
```

Comment:
```suggestion
                            SelectedChipView("일반 카테고리")
```
아무래도 네이밍은 잡고 가야할거 같아요. 다시 코드 볼때마다 액션으로 착각하고 구현부를 찾아보게 되네요

Reply:
https://github.com/ku-ring/ios-app/commit/dfb9a1305752ceccd7c4ba51682176b7445aed8b